src/data/dem_tree.js

```diff
@@ -177,6 +177,8 @@ export default class DemMinMaxQuadTree {
         this._construct(mips, 0, 0, maxLvl, 0);
     }
 
+    get rootBounds(): [number, number] { return [-aabbSkirtPadding, this.maximums[0]]; }
```

Comment:
Should this be `this.minimums[0] - aabbSkirtPadding` in case the terrain is below sea level? Or maybe `Math.min(this.minimums[0], 0) - aabbSkirtPadding` if the goal is to get it to intersect with elevation = 0?